content/en/cosign/working_with_blobs.md

```diff
@@ -40,44 +40,7 @@ cosign sign --key cosign.key gcr.io/user-vmtest2/artifact
 Enter password for private key:
 Pushing signature to: gcr.io/user-vmtest2/artifact:sha256-3f612a4520b2c245d620d0cca029f1173f6bea76819dde8543f5b799ea3c696c.sig
 ```
-### sget
```

Comment:
Is this being removed because of sget's testing status? Could you provide more details in a related issue?

Reply:
I recommended this because sget in Cosign is deprecated (https://github.com/sigstore/cosign/blob/main/cmd/sget/main.go#L34) and the sget repo is marked as experimental. We shouldn't include this in the documentation until it's stable.